tests/tests/constants/compiledContracts/finiteloopcontract.json

```diff
@@ -0,0 +1,2128 @@
+{
```

Comment:
I thought we agreed on file name like: `finite-loop-contract.json`

Reply:
See https://github.com/PureStake/moonbeam/pull/351#discussion_r614757741
I could do that but then I would need to add a module to convert form CamelCase to kebab-case for the file name

Reply:
Ok